naturf/nodes.py

```diff
@@ -1514,21 +1487,13 @@ def raupach_roughness_length(
     :return:                              Pandas DataFrame with Raupach roughness length in each cardinal direction.
     """
 
-    raupach_roughness_length_north = Settings.raupach_roughness_length_north
-    raupach_roughness_length_east = Settings.raupach_roughness_length_east
-    raupach_roughness_length_south = Settings.raupach_roughness_length_south
-    raupach_roughness_length_west = Settings.raupach_roughness_length_west
-
     cols = [
-        raupach_roughness_length_north,
-        raupach_roughness_length_east,
-        raupach_roughness_length_south,
-        raupach_roughness_length_west,
+        Settings.raupach_roughness_length_north,
+        Settings.raupach_roughness_length_east,
+        Settings.raupach_roughness_length_south,
+        Settings.raupach_roughness_length_west,
     ]
 
-    cols_fai = frontal_area_index.columns
-    cols_rdh = raupach_displacement_height.columns
-
```

Comment:
When raupach_roughness_length() is run, it changes the column names for frontal_area_index and raupach_displacement_height, which prevents merge_parameters from working correctly. These lines were an attempt to prevent the input dataframes from being modified, but is there a better way to do that? Maybe assigning new variables i.e. `fai = frontal_area_index` and `rdh = raupach_displacement` and then using those variables in the calculation?

Reply:
gotcha, I'll leave that in